### ISSUE TITLE
AdditionalSongFolders metric doesn't work, gets reset

### ISSUE THREAD

**Issue by JapanYoshi** [stepmania/stepmania#1986]
Version: 5.1.0-b2

# Steps to reproduce:

1. Go to \Save\Preferences.ini
2. Set AdditionalSongFolders to an existing folder path
3. Run Stepmania
4. Reload all songs and courses

# Expected result:

Songs are loaded from that folder.

# Actual result:

No songs are loaded from that folder, and the preference AdditionalSongFolders is set to an empty string.

**Comment by quietly-turning** [comment]
Thanks for submitting such nicely formatted, helpful bug report. 🙂 

I tested this using a fresh build of SM5.1 (up-to-date as of 8650fc420e747fb95d98610200ac3ee7ec23dfab) and `AdditionalSongFolders` worked okay for me. 

I provided two different paths separated by a comma, and both paths were picked up and the songs were playable when I ran StepMania.  My line looked like
```ini
AdditionalSongFolders=/home/quietly/Desktop/MoreSongs,/home/quietly/StepManiaStuff/OtherSongs
```
In this example, `MoreSongs` contained multiple sub-folders of groups, and each group contained multiple sub-folders of songs.  `OtherSongs` followed the same pattern.

If you're sure that your folder structure matches what I've described and are still having this issue, respond with more specific information: 
 * where you have StepMania installed
 * what your `AdditionalSongFolders` line looks like
 * the git hash of the SM5.1 build you are using

The git hash should appear in the upper-right corner of the Title screen in SM5.1's default theme like this:
![Screen Shot 2020-04-19 at 8 27 34 AM](https://user-images.githubusercontent.com/1253483/79687798-c76eaf80-8217-11ea-86dc-cccddc81d5d6.png)
